scalableminds/webknossos
# Make active tree node round instead of shifting the hue value

### URL of deployed dev instance (used for testing):
- https://roundactivenode.webknossos.xyz

### Steps to test:
- open a skeleton tracing and change the active node (e.g., create new nodes)
- the active node should be round and should have the same color as the other nodes of the same tree

### Issues:
- fixes #3575 

------
